packages/react-components/src/__tests__/mail.test.ts

```diff
@@ -6,8 +6,18 @@ import {
 } from '../mail';
 
 describe('createMailTo', () => {
-  it('generates a mailto link', () => {
-    expect(new URL(createMailTo('test@example.com')).protocol).toBe('mailto:');
+  it('generates a mailto link, when an single email is provided', () => {
```

Comment:
We should be testing an array with one item

Reply:
Now we are testing both 3 cases